presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -758,7 +759,14 @@ public void testParseDatetime()
     public void testFormatDatetime()
     {
         assertFunction("format_datetime(" + TIMESTAMP_LITERAL + ", 'YYYY/MM/dd HH:mm')", VARCHAR, "2001/08/22 03:04");
-        assertFunction("format_datetime(" + TIMESTAMP_LITERAL + ", 'YYYY/MM/dd HH:mm ZZZZ')", VARCHAR, "2001/08/22 03:04 Asia/Kathmandu");
+        if (isLegacyTimestamp(session)) {
```

Comment:
I thought that tests for legacy time related types were moved to `Test*Legacy`, shouldn't be this one moved (split) as well?